rampwf/workflows/simplified_image_classifier.py

```diff
@@ -83,6 +84,16 @@ def test_submission(self, trained_model, folder_X_array):
         return y_proba
 
 
+def image_transform(x, transforms):
```

Comment:
@mehdidc @jorisvandenbossche I'll need code review here. Remember, the problem is that because of `imp` we can't pickle any user submission, which means they cannot implement parallel loading or image transform. Implementing loading was easy, but for the transform they also can't provide the transform function itself, so we need to implement the transform itself, which they can parameterize. The syntax is a list of dictionaries, each dictionary containing `name` and a list of named parameters. Below is the implementation of a random rotation between `l_angle` and `u_angle`.